azerothcore/azerothcore-wotlk
# fix(Core/ZulGurub): Hakkar hardmode

## Changes Proposed:
-  Modified hardmode abilities according to the issue.
-  Hakkar should use Hakkar Power and Hakkar Power Down to modify his scale, hp and damage.

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/12287

## SOURCE:


## Tests Performed:

- Tested in-game (not spell timers)

## How to Test the Changes:


1. Go to Zul Gurub and check Hakkar hp with different combinations of  priests alive and killed.
2. Engage Hakkar with all priests alive and check the timers.
